gen/src/write.rs

```diff
@@ -1753,12 +1807,18 @@ fn write_rust_vec_impl(out: &mut OutFile, key: NamedImplKey) {
 }
 
 fn write_rust_option_impl(out: &mut OutFile, inner: OptionInner) {
+    let mut value_needs_ref = false;
+    let mut is_const = false;
     let element = match inner {
```

Comment:
Same as prior, I'd prefer if each branch in the match fully provided `value_needs_ref` and `is_const`, so it doesn't require looking in two places to understand the value.